### ISSUE TITLE
use distribution package throw command not found

### ISSUE THREAD

**Issue by JennerChen** [vadimdemedes/pastel#12]
I just try to use pastel write a demo cli project. After I publish package and install&run it on my target server. It throw error below. 

```
/Users/0004112/temp/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

Error: Cannot find module '/Users/zhangqing/Documents/web/pastel-demo/build/commands/index.js'
```

I found the bug is located in the generated commands.json file , command.path is my dev machine path. 

```
{
	"commands": [
		{
			"path": "/Users/zhangqing/Documents/web/pastel-demo/commands/index.js",
			"buildPath": "/Users/zhangqing/Documents/web/pastel-demo/build/commands/index.js",
			"name": "index",
			"description": "This is my command description",
			"args": [
				{
					"key": "name",
					"type": "string",
					"description": "This is \"name\" option description",
					"isRequired": true,
					"aliases": [
						"n"
					],
					"positional": false
				}
			],
			"subCommands": []
		},
		{
			"path": "/Users/zhangqing/Documents/web/pastel-demo/commands/posts/index.js",
			"buildPath": "/Users/zhangqing/Documents/web/pastel-demo/build/commands/posts/index.js",
			"name": "posts",
			"description": "",
			"args": [],
			"subCommands": []
		}
	]
}
```

I am new to pastel, I don't know whether is my mistake or it is a bug in pastel. 
here is my demo package: 
npm:https://www.npmjs.com/package/@mogul/hello-person-demo 
git: https://github.com/JennerChen/try-pastel-person-demo
env:
nodejs: 8.12.0
pasteljs: 1.0.1

**Comment by MihaiVoinea** [comment]
I also had this issue with my package.

https://github.com/mihaivoinea/makerlog-cli

I used create-pastel-app to generate the boilerplate.

**Comment by vadimdemedes** [comment]
Good catch! Will publish a fix soon.

**Comment by vadimdemedes** [comment]
Released a fix as `1.0.2`. You need to rebuild and publish a new version of your CLI to fix this issue.
